scripts/log-templates/update-log-config.yaml

```diff
@@ -1,20 +1,9 @@
-# Same as log-template.yaml, but without files, 
+# Same as log-template-demo.yaml, but without files,
```

Comment:
Without files? What does it mean? It has files output. I guess the difference is in rotation.

Reply:
I have no idea, I am not competent enough to clarify this comment.